### ISSUE TITLE
Proper style for input-group-addon

### ISSUE THREAD

**Issue by 19Gerhard85** [mdbootstrap/mdb-ui-kit#125]
Hello Fez,

would be nice if input-group-addon has a more matching look to the other elements.

e.g. 
![input-group-addon](https://cloud.githubusercontent.com/assets/1492978/4679105/f6f47024-55fd-11e4-9315-61131ef1ca29.png)

Left: How it could look like
Right: As it looks right now

Greets

**Comment by FezVrasta** [comment]
I remember I've did this work already in past... Maybe I've overwritten it later, will try to do something asap

**Comment by FezVrasta** [comment]
Looking at the demo everything looks okay...

![image](https://cloud.githubusercontent.com/assets/5382443/4689857/36d4d8fa-56d0-11e4-8faa-9f2b60801748.png)

Am I missing something?

**Comment by 19Gerhard85** [comment]
I've tried the code from your demo page, but I my case it looks different.

You can checkout the jsfiddle: http://jsfiddle.net/ftnwtdvq/3/

``` html
<div class="well bs-component">
    <fieldset>
        <legend>Legend</legend>
        <div class="row">
            <div class="form-group col-lg-4">
                <label for="grundfl">Label</label>
                <div class="input-group">
                    <input type="text" class="form-control" id="grundfl" value="123">
                    <span class="input-group-addon">xx</span>
                </div>
            </div>
        </div>
    </fieldset>
</div>
```

**Comment by destan** [comment]
@19Gerhard85 Your fiddle looks like this for me:
![screenshot from 2014-10-20 11 51 05](https://cloud.githubusercontent.com/assets/442813/4698583/50141580-5836-11e4-9717-7ef016d4e36f.png)

`Chromium Version 37.0.2062.120 Ubuntu 14.04 (281580) (64-bit)`

**Comment by FezVrasta** [comment]
Will look into this..

**Comment by 19Gerhard85** [comment]
@destan
Looks the same for me.

@FezVrasta 
Thanks!

**Comment by FezVrasta** [comment]
lol, it had background #EEE and the demo has as brackground #EEE so it was hidden. Fixed with 9299bda
